aws/resource_aws_api_gateway_integration.go

```diff
@@ -89,15 +97,19 @@ func resourceAwsApiGatewayIntegration() *schema.Resource {
 			"content_handling": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				ValidateFunc: validateApiGatewayIntegrationContentHandling,
+				ValidateFunc: validateApiGatewayIntegrationContentHandling(),
 			},
 
 			"passthrough_behavior": {
-				Type:         schema.TypeString,
-				Optional:     true,
-				Computed:     true,
-				ForceNew:     true,
-				ValidateFunc: validateApiGatewayIntegrationPassthroughBehavior,
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"WHEN_NO_MATCH",
+					"WHEN_NO_TEMPLATES",
+					"NEVER",
+				}, false),
```

Comment:
Is there any particular reason you decided to make this validation inline instead of keeping it decoupled like all the others?

Reply:
My goal is to make validation inline as default in case of functions in `validation` package. The exception is only when it's used exactly the same by multiple arguments, like `validateHTTPMethod()` and `validateApiGatewayIntegrationContentHandling()` in this PR.

Reply:
Oh, I see - reusability, doh! Obviously - good point! 👍 😄